package.json

```
@@ -22,7 +22,7 @@
     "test-stream": "npm run mocha test/stream.test.ts",
     "test-media": "npm run mocha test/media-upload.test.ts",
     "test-auth": "npm run mocha test/auth.test.ts",
-    "prepublish": "npm run build"
+    "prepublish": "yarn build"
```

Typo I guess, the project uses npm :)
```suggestion
    "prepublish": "npm run build"
```